app/build.gradle

```diff
@@ -35,6 +35,8 @@ android {
 
     defaultConfig {
         applicationId versions.packageName
+        versionCode 10701
```

Comment:
So the tag will be on the commit preceding this one?

Reply:
Yep, the tag is on f0923250dcc2527da3ae96a0386e02edab24e381